namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-test/resources/rds.tf

```diff
@@ -12,7 +12,11 @@ module "rds" {
   db_engine_version            = "14"
   db_instance_class            = "db.t3.small"
   rds_family                   = "postgres14"
-  allow_major_version_upgrade  = "true"
+  allow_major_version_upgrade  = "false"
+  allow_minor_version_upgrade  = "true"
+
+  backup_window                = var.rds_backup_window
```

Comment:
I borrowed the configuration of these from [Locations Inside Prison](https://github.com/ministryofjustice/cloud-platform-environments/blob/8da1196b0ccec1912a8877b8f8e2fc2177aec759/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-locations-inside-prison-prod/resources/rds.tf#L19).